setup.cfg

```diff
@@ -57,11 +57,12 @@ install_requires =
     pandas
     pandasql
     pyyaml
-    rdflib
+    rdflib==5.0.0
+    rdflib-jsonld==0.6.0
     recommonmark>=0.7
     scikit_learn
     scipy
-    setuptools
+    setuptools<58.0
```

Comment:
is this still necessary?

Reply:
Yes.

Reply:
Why? Just so we have it documented :) thank you!

Reply:
My bad, it is not needed. Took it off.